frontend/package-lock.json

```diff
@@ -12304,27 +12309,6 @@
         "@wry/context": "^0.4.0"
       }
     },
-    "optimist": {
```

Comment:
Here's another removed section that may need to stay.  @michaelwoodruffdev ?

Reply:
The "optimist": {

Reply:
I will defer to @michaelwoodruffdev / @gabe-heim